### ISSUE TITLE
advisory

### ISSUE THREAD

**Issue by gengyanlei** [tau-yihouxiang/WS_DAN#9]
Author, hello, may I ask you to use the tensorflow version?
thanks

**Comment by tau-yihouxiang** [comment]
I have tested it in TensorFlow 1.6 and TensorFlow 1.14.

**Comment by gengyanlei** [comment]
thanks,and I have tested successfully.

**Comment by gengyanlei** [comment]
Hello, author.
When I read your code, I found a problem, I can't understand it, can you explain it, can you? Thank you.
The question is: lines 561 and 562 in the code train_sample.py on github, why is it duplicated? What is the role?
Code:  logits_2, end_points_2 = network_fn(images_crop, reuse=True)

**Comment by QingxinWx** [comment]
Hello, can I ask you one question?
After I run eval_sample_bird.sh , I will get a file named events.out.tfevents.1568983007.lthpc.
How can I continue to get the accuracy or other result？
thanks！！
